README.md

```diff
@@ -146,3 +147,73 @@ prefixed with the attribute `#[verbose]` to get an idea of what they are doing.
   this seamingly mutual dependency using HB.
 
 </p></details>
+
+### Plan B
+
+Scared of making your project depend on HB? This section is for you.
+
+HB is based on a thick layer of software which we plan to maintain, but we
+also understand it can look scary. Hence this insurance plan. By passing
+the attribute `#[log]` each command prints Coq commands which are equivalent to
+its effect. By replacing each HB command by its equivalent Coq commands, you
+can eliminate the dependency on HB from your project.
+
+This is a "plan B", by looking at the output of`#[log]` you will realize that
+HB commands are much nicer (and shorter) than the equivalent Coq code. The
+point of a "plan B" is to avoid nightmares, not to be nicer than plan A ;-)
+
+How can you be ensure plan B works? We provide tools to check that, see
+the details below.
+
+<details><summary>(click to expand)</summary><p>
+
+
+Hierarchy Builder commands can log their equivalent vernacular commands
+to "patch" file (extension `.hb`). In order to do so, one has to
+compile the project with the `COQ_ELPI_ATTRIBUTES` variable set. Eg
+
+```shell
+COQ_ELPI_ATTRIBUTES='hb(log(raw))' make
```

Comment:
Is this idiomatic or can I subtly alter this variable to get different results?

Reply:
FTR: `COQ_ELPI_ATTRIBUTES=foo` passes to all commands written in Elpi `#[elpi(foo)]` and all attributes named `elpi.*` are silently ignored (instead of being errors) if not supported by the command. This has nothing to do with HB. If the command has other attributes, they are concatenated, eg `#[verbose, elpi(foo)]`. All commands always get `elpi.loc="location of the command"` since coq-elpi 1.9.

Hence, if you pass `COQ_ELPI_ATTRIBUTES='hb(log(raw))'` you pass `#[elpi(hb(log(raw))]` which has the same effect of `#[log(raw)]` but prints to a patch file.
If you pass `COQ_ELPI_ATTRIBUTES='hb(log)'` things work but the patches are not guaranteed to be reparsable, as it happens with the text you get by passing `#[log]`.

The `hb` part is not needed, strictly speaking, but seems good hygiene since the env variable is "global" to all commands written in Elpi, not just HB.